apps/signal/CMakeLists.txt

```diff
@@ -18,6 +18,14 @@ config_option(
     DEPENDS
     "DefaultBenchDeps"
 )
+config_option(
+    AppSignalEarlyProcessing
+    APP_SIGNAL_EARLYPROC
+    "Apply early processing of the raw results for Signal benchmark"
```

Comment:
I think we need to clarify what "early processing" means somewhere (README?) - basically one option gives you exact measurements for each measurement but may have more cache effects vs calculating stats on the fly.

Reply:
The option has been removed.